<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tcp</class>
 <widget class="QWidget" name="Tcp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>680</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1040</width>
    <height>680</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1040</width>
    <height>680</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tcp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(16,16,16)</string>
  </property>
  <widget class="QLabel" name="Jwlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>320</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48,49,51)</string>
   </property>
   <property name="text">
    <string>机位控制</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="JWwidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>112</y>
     <width>320</width>
     <height>368</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>368</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>368</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48,49,51)</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>132</x>
      <y>21</y>
      <width>14</width>
      <height>10</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>14</width>
      <height>10</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>14</width>
      <height>10</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>2</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labeltext">
    <property name="geometry">
     <rect>
      <x>152</x>
      <y>20</y>
      <width>37</width>
      <height>13</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>37</width>
      <height>13</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>37</width>
      <height>13</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>248</x>
      <y>16</y>
      <width>52</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>52</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>52</width>
      <height>20</height>
     </size>
    </property>
    <property name="text">
     <string>切换场景</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>320</width>
      <height>118</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>118</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>118</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea_5">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>320</width>
          <height>118</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_3">
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <property name="verticalSpacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="SWitchwidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>499</y>
     <width>320</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>160</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>160</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48,49,51)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea_3">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>320</width>
         <height>160</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="BigVediolabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>320</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48,49,51)</string>
   </property>
   <property name="text">
    <string>大荧幕</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="CenterVediowidget" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>112</y>
     <width>320</width>
     <height>296</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>296</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>296</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48,49,51)</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QLabel" name="Fwabel">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>428</y>
     <width>320</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48,49,51)</string>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="text">
    <string>氛围切换</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QWidget" name="SWitchwidget_2" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>520</y>
     <width>320</width>
     <height>140</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>140</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>140</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48,49,51)</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>320</width>
         <height>140</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="horizontalSpacing">
         <number>80</number>
        </property>
        <property name="verticalSpacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="HDlabel">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>20</y>
     <width>320</width>
     <height>90</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>90</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48,49,51)</string>
   </property>
   <property name="text">
    <string>互动道具</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="HDwidget" native="true">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>112</y>
     <width>320</width>
     <height>548</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>548</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>320</width>
     <height>548</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(48,49,51)</string>
   </property>
   <widget class="QScrollArea" name="HDscrollArea">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>320</width>
      <height>490</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>490</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>320</width>
      <height>490</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="HDscrollAreaWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>320</width>
       <height>490</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="MusicLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>98</width>
      <height>12</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>98</width>
      <height>12</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>98</width>
      <height>12</height>
     </size>
    </property>
    <property name="text">
     <string>音乐可视化强度</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>250</width>
      <height>15</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="maximum">
     <number>31</number>
    </property>
    <property name="singleStep">
     <number>0</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_slider">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>27</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>30</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
